stock_landed_cost_company_percentage/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_landed_cost_company_line_wh_manager,access_landed_cost_company_line,model_landed_cost_company_line,stock.group_stock_manager,1,1,1,1
+access_landed_cost_company_line_wh_user,access_landed_cost_company_line,model_landed_cost_company_line,stock.group_stock_user,1,0,0,0
+access_landed_cost_company_line_user,access_landed_cost_company_line,model_landed_cost_company_line,base.group_user,1,0,0,0
```

Comment:
you also want a record rule to segregate in multicompany

Reply:
Corresponded. Thank you.